espanol/huida/trabajo.md

```diff
@@ -0,0 +1 @@
+El cocinero dijo que no le gusta cocinar para muchas personas.
```

Comment:
It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.